server/graphql/v1/mutations/collectives.js

```diff
@@ -732,8 +732,13 @@ export async function deleteCollective(_, args, req) {
     throw new Error('Can not delete collective with paid expenses.');
   }
 
+  const eventCollectiveIds = await models.Collective.findAll({
+    attributes: ['id'],
+    where: { ParentCollectiveId: collective.id },
```

Comment:
`ParentCollectiveId` can have other uses, we need to double check with the type

```suggestion
    where: { ParentCollectiveId: collective.id, type: types.EVENT },
```